modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2955,6 +2956,16 @@
 		end
 	end
 
+	function m.useStandardPreprocessor(cfg)
+		if _ACTION >= "vs2019" and cfg.usestandardpreprocessor ~= nil then
+			if cfg.usestandardpreprocessor == 'On' then
+				m.element("UseStandardPreprocessor", nil, "true")
+			else
```

Comment:
We force as "On" or "Off".  What happens if we get a value that isn't "On" or "Off"?

Reply:
It would just get defaulted to false, which is the default option on msvc anyways. I could add a check for it and error if that'd be better

Reply:
@nickclark2016 it shouldn't be possible to get a value that isn't `On` or `Off` due to the API having an `allowed` list with only those two values.